dadi/lib/datasource/index.js

```diff
@@ -1,20 +1,22 @@
 /**
  * @module Datasource
  */
-var _ = require('underscore')
-var fs = require('fs')
-var path = require('path')
-var url = require('url')
+const _ = require('underscore')
```

Comment:
Could we use this as an opportunity to get rid of Underscore on this file? 😬 Only 3 instances I can see.

Reply:
Absolutely!

Reply:
🎉

Reply:
#198

Reply:
That issue should have a list of files that still contain underscore usage :)